### DISCUSSION TITLE
Missing information in documentation of `deploy`

### DISCUSSION THREAD

**Discussion by algomaster99** [jreleaser/jreleaser#1342] [Q&A] [answered]
Hi! I had a difficulty finding the documentation for configuring deployment. For example, I have this [deployment configuration](https://github.com/ASSERT-KTH/sorald/blob/master/jreleaser.json#L58-L72) for my project. I tried to look for ways [here](
https://jreleaser.org/guide/latest/reference/deploy/index.html), initially. Eventually, I had to find this out from different projects, such as https://github.com/chains-project/maven-lockfile/blob/main/jreleaser.yml.

**Comment by aalmiray** [comment, marked as answer]
The guide provides examples for both Maven and Gradle https://jreleaser.org/guide/latest/examples/maven/maven-central.html

**Reply by algomaster99** [reply]
Oh right. You shared its link yesterday. :sweat_smile: Thanks again!
